File: secao-11-Linguagem-Java/exercicios/soma_vetor.java
// Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida: 
// - Imprimir todos os elementos do vetor 
// - Mostrar na tela a soma e a média dos elementos do vetor

import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int N, i;
        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        double[] vet = new double[N];
        for (i = 0; i < N; i++) {
            vet[i] = sc.nextDouble();
        }

        System.out.print("Valores = ");
        for (i = 0; i < N; i++) {
            System.err.print(String.format("%.1f", vet[i]) + " ");
        }

        double soma = 0;
        for (i = 0; i < N; i++) {
            soma = soma + vet[i];
        }

        double media = soma / 2.0;

        System.out.println();
        System.out.println();
        System.out.println("Soma = " + String.format("%.2f", soma));
        System.out.println("Media = " + String.format("%.2f", media));

        sc.close();
    }
}

File: secao-13-Linguagem-Python/exercicios/soma_vetor.py
# Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida:
# - Imprimir todos os elementos do vetor
# - Mostrar na tela a soma e a média dos elementos do vetor

N = int(input("Quantos números você vai digitar: "))

vet = [0 for x in range(N)]
for i in range(0, N):
    vet[i] = float(input("Digite um número: "))

soma = 0
for i in range(0, N):
    vet[i] = soma + vet[i]

media = soma / N

print()
print("Valores = ", end="")
for i in range(0, N):
    print(f"{vet[i]:.1f} ", end="")

print()
print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")